cdap-hbase-compat-base/src/main/java/co/cask/cdap/replication/ReplicationStatusKey.java

```diff
@@ -47,43 +57,38 @@ public ReplicationStatusKey(byte[] key) {
     return key;
   }
 
-  /**
-   * Splits the keys into a {@link Splitter} which exposes the parts that comprise the key.
-   */
-  public Splitter split() {
-    return new Splitter(key);
+  public String getRowType() {
+    return rowType;
   }
 
-  /**
-   * Decodes the keys parts
-   */
-  public static final class Splitter {
-    private final ByteBuffer byteBuffer;
+  public String getRegionName() {
+    return regionName;
+  }
 
-    private Splitter(byte[] bytes) {
-      byteBuffer = ByteBuffer.wrap(bytes);
-    }
+  public String getRsID() {
+    return rsID;
+  }
 
-    /**
-     * @return the next byte[] part in the splitter
-     * @throws BufferUnderflowException if there is no byte[] as expected
-     */
-    public byte[] getBytes() {
-      int len = byteBuffer.getInt();
-      if (byteBuffer.remaining() < len) {
-        throw new BufferUnderflowException();
-      }
-      byte[] bytes = new byte[len];
-      byteBuffer.get(bytes, 0, len);
-      return bytes;
+  /**
+   * @return the next byte[] part in the splitter
+   * @throws BufferUnderflowException if there is no byte[] as expected
+   */
+  private byte[] getBytes(ByteBuffer byteBuffer) {
+    int len = byteBuffer.getInt();
+    if (byteBuffer.remaining() < len) {
+      throw new BufferUnderflowException();
     }
+    byte[] bytes = new byte[len];
+    byteBuffer.get(bytes, 0, len);
+    return bytes;
+  }
 
-    /**
-     * @return the next String part in the splitter
-     * @throws BufferUnderflowException if there is no String as expected
-     */
-    public String getString() {
-      return Bytes.toString(getBytes());
-    }
+  /**
+   * @return the next String part in the splitter
+   * @throws BufferUnderflowException if there is no String as expected
+   */
+  private String getString(ByteBuffer byteBuffer) {
```

Comment:
in general, one line methods don't do anything except add unnecessary complexity. Just call Bytes.toString() instead of this.